examples/advanced/sklearn-kmeans/src/kmeans_trainer.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
2025
Personally preference suggest rename this file to "kmeans_fl.py"

Reply:
good idea, updated